### ISSUE TITLE
USER ISSUE: Frame-Crash (?)

### ISSUE THREAD

**Issue by Siesma** [StrangeLoopGames/EcoIssues#14176]
**Version:** 0.8.3.1 beta

**Steps to Reproduce:**
For me the game always crashes (and captures the complete screen so I have to logout of windows in order to close ECO) when I receive a new Notification of any kind. 

For example:
Someone sends me a message via Telegram. Telegram offers "pop-up-notifications" and these cause my ECO to freeze.

Technical information:
First time occured: ECO update 0.8.3.1
CPU: I7-4790k
GPU: Nvidia Geforce GTX 980
OS: Windows 7 professional

**Expected behavior:**
The game should just continue and not try to go into the background.

**Actual behavior:**
Instead it crashes and becomes "un-clickable".

**Do you have mods installed? Does issue happen when no mods are installed?:**
I have installed the "Ore-Washer" mod, although I doubt, that this is the cause.

**Comment by mirasrael** [comment]
@Siesma which display mode selected for Eco in graphics settings?

**Comment by SlayksWood** [comment]
1. Do you have Firewall or Antivirus enabled?
2. Did you meet this issue after updating to 0.8.3.1 or you had it before?
3. Please provide log files
Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by SlayksWood** [comment]
Step to reproduce:
- start game, join to any server.
- open Graphics settings and select Exclusive Fullscreen, Confirm
![image](https://user-images.githubusercontent.com/45708377/71154474-9de34e80-224c-11ea-85e2-2a3eedc19fe5.png)
- minimize game and then restore it.
- game freezes.

**Comment by SlayksWood** [comment]
@Siesma 
You can use Bordeless Fullsceen until this issue is fixed.

**Comment by Siesma** [comment]
First of all, sry for taking so long.

mirasrael:
1: I am using exclusive Fullscreen at 1920x1080

SlaysWood:
1: Yes, but I also tried it without one, still the same crash occured
2: This bug "came" with the 0.8.3.1 update. Before (0.8.3.0) was no crashing related to new window changes.
3: Added (hopefully)

I will try and use the borderless fullscreen mode.
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/3984798/Player.log)
